doc/instruction_decode_execute.rst

```diff
@@ -64,10 +64,31 @@ Other blocks use the ALU for the following tasks:
 * It computes memory addresses for loads and stores with a Reg + Imm calculation
 * The LSU uses it to increment addresses when performing two accesses to handle an unaligned access
 
-Support for the RISC-V Bitmanipulation Extension (Document Version 0.92, November 8, 2019) is enabled via the parameter ``RV32B``.
-This feature is *EXPERIMENTAL* and the details of its impact are not yet documented here.
-Currently the Zbb, Zbs, Zbp, Zbe, Zbf, Zbc, Zbr and Zbt sub-extensions are implemented.
-The rotate instructions `ror` and `rol` (Zbb), ternary instructions `cmov`, `cmix`, `fsl` and `fsr` as well as cyclic redundancy checks `crc32[c]` (Zbr) are completed in 2 cycles. All remaining instructions complete in one cycle.
+Bit Manipulation Extension
+  Support for the RISC-V Bitmanipulation Extension (Document Version 0.92, November 8, 2019) is enabled via the enumerated parameter ``RV32B`` defined in :file:`rtl/ibex_pkg.sv`. This feature is *Experimenal*.
```

Comment:
A link to the mentioned document would also be nice.

Reply:
Yes, done.